org.gaphor.Gaphor.yaml

```diff
@@ -0,0 +1,55 @@
+app-id: org.gaphor.Gaphor
+runtime: org.gnome.Platform
+runtime-version: '3.30'
+sdk: org.gnome.Sdk
+rename-icon: gaphor
+command: gaphor
+finish-args:
+- "--socket=x11"
+- "--socket=fallback-x11"
+- "--share=ipc"
+- "--socket=wayland"
+- "--persist=.gaphor"
```

Comment:
Ideally it should be respecting `$XDG_CONFIG_DIR` instead.

Reply:
I updated Gaphor upstream to use XDG_DATA_HOME instead of ~/.gaphor. Thanks for the tip, I didn't know about that standard.